src/test/java/blackjack/domain/profit/PlayersProfitTest.java

```diff
@@ -0,0 +1,68 @@
+package blackjack.domain.profit;
+
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Player;
+import blackjack.domain.prize.PlayersPrize;
+import blackjack.domain.table.BettingTable;
+import blackjack.dto.PlayerInput;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static blackjack.domain.fixture.TestCards.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PlayersProfitTest {
+    @Test
+    @DisplayName("베팅테이블과 플레이어들의 승패결과가 주어지면 플레이어들의 이익을 반환한다.")
+    void generate() {
+        // given
+        BettingTable bettingTable = generateBettingTable();
+        PlayersPrize playersPrize = generatePlayersPrize();
+
+        // when
+        PlayersProfit playersProfit = new PlayersProfit(bettingTable, playersPrize);
+        List<PlayerProfit> profits = playersProfit.getPlayerPrizes();
+
+        // then
+        assertThat(profits).hasSize(3);
+        assertThat(profits.get(0).getPlayerName()).isEqualTo("a");
+        assertThat(profits.get(1).getPlayerName()).isEqualTo("b");
+        assertThat(profits.get(2).getPlayerName()).isEqualTo("c");
```

Comment:
```suggestion
        assertThat(profits).hasSize(3)
                .extracting("playerName")
                .containsExactly("a", "b", "c");
```
요렇게 할 수 있어요 :)
(순서가 상관 없는 검증이라면 `containsExactlyInAnyOrder` 를 사용할 수도 있구요.
검증할 값이 여러 개라면 `tuple`을 사용할 수도 있어요. 한번 찾아보시면 좋을 것 같아요 ㅎㅎ)

Reply:
오 `AssertJ`에 유용한 메서드들이 많네요! 감사합니당